tests/lib/testing_classes.py

```diff
@@ -129,6 +129,9 @@ def env_var(self, monkeypatch_session, download_test_data):
             monkeypatch_session.setenv(key, str(value))
 
         #reset connection to openpype DB with new env var
+        if mongo_url:
```

Comment:
shouldn't it be to `monkeypatch_session.setenv` instead?

Reply:
And on line 156 getting it from `monkeypatch_session` (I think there was a bug there originally, potentially - with `os.environ`.)

Reply:
Yes, setting `monkeypatch_session.setenv(key, str(value))` will set env var 'key' with 'value' automatically.